front/src/views/Boxes/components/BoxesActionsAndTable.tsx

```diff
@@ -221,6 +236,38 @@ function BoxesActionsAndTable({
         selectedBoxes={selectedBoxes}
         key="remove-boxes"
       />,
+      <Button
+        onClick={() => {
+          exportBoxData();
+        }}
+        key="export"
+        data-testid="export-button"
+      >
+        <CSVLink
+          data={exportData}
+          filename="Stock Overview"
```

Comment:
In dropapp, the filename contains date and timestamp, e.g. `Stock_2024-11-25_083811.csv` if that's still relevant